src/comms.h

```diff
@@ -12,4 +12,6 @@ void close_door();
 
 void set_lock(uint8_t value);
 void set_light(bool value);
+
+void led_loop();
```

Comment:
This shouldn't go in comms.h.  led_loop() is defined in ratgdo.cpp, and is only referenced in ratgdo.cpp, so just move the declaration to the top of ratgdo.cpp

If you needed to call it from another cpp file, then you would put this in the ratgdo.h file